services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -100,13 +100,16 @@ export const AddEditReportModal = ({
         state: activeState,
         id: selectedReport.id,
       };
-
       // edit existing report
       await updateReport(reportKeys, {
         ...dataToWrite,
         metadata: {
           ...dataToWrite.metadata,
-          status: ReportStatus.IN_PROGRESS,
+          locked: undefined,
+          status: reportType !== "MLR" ? ReportStatus.IN_PROGRESS : undefined,
+          submissionCount: undefined,
+          previousRevisions: undefined,
+          // status: selectedReport?.submissionCount >= 1 ? ReportStatus.IN_PROGRESS : `In revision`,
```

Comment:
Please validate if the commented out code is needed or can be deleted

Reply:
Should be removed!